syntax = "proto3";

package order;

service OrderService {
  rpc CreateOrder (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
  int32 user_id = 1;
  int32 product_id = 2;
  int32 quantity = 3;
}

message OrderResponse {
  string message = 1;
}
